site2/docs/admin-api-brokers.md

```diff
@@ -35,7 +35,7 @@ Pulsar brokers consist of two components:
 
 In addition to being configurable when you start them up, brokers can also be [dynamically configured](#dynamic-broker-configuration).
 
-For a full listing of broker-specific configuration parameters, see the [Configuration](reference-configuration.md#broker) page.
+For a full listing of broker-specific configuration parameters, see the [Configuration](https://pulsar.apache.org/reference/#/config/reference-configuration-broker) page.
```

Comment:
This URL can not be used since we're working in progress and it might be changed later, right? @SignorMercurio

Reply:
Right.